monitoring/ping_healthchecks.py

```diff
@@ -0,0 +1,25 @@
+import os
+import logging
+import socket
+import requests
+
+LOGGER = logging.getLogger(__name__)
+
+HEALTH_CHECK_URL_ENV = "HEALTH_CHECK_URL"
+DEFAULT_HEALTH_CHECK_URL_ENV = "https://hc-ping.com/3549bc53-4bd9-4ef4-9c4a-22aa57c2fb5b"
```

Comment:
minor name: probably should be `DEFAULT_HEALTH_CHECK_URL` (without the `_ENV`)

Is this a development healtheck url and we will use a different one in production?